### PR TITLE
Added ComMatrix method; Changed private func to public

### PR DESCRIPTION
**Pull request by shirmoran**
This PR changed functions in order to use them in a new [openshift\origin](https://github.com/openshift/origin) test specified [here](https://issues.redhat.com/browse/CNF-12146) : 
Changed "addFromFile" function (in types.go) to a public function to allow using it in the mentioned above test.

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -160,3 +160,36 @@ func (m ComMatrix) Contains(cd ComDetails) bool {
 
 	return false
 }
+
+// Returns True if both ComMatrixes have exactly the same ComMatrix details.
+func (m ComMatrix) Equals(other ComMatrix) bool {
```
Comment (aabughosh) [inline types/types.go:165]: why not to use the Diff function?
u need from diff to be equal to nothing from 2 sides

Comment (aabughosh) [reply types/types.go:165]: @sabinaaledort also can use that on the test of the origin

Comment (shirmoran) [reply types/types.go:165]: Because we don't need the Diff matrices is this case, it occupies unnecessary space.

**Comment by sabinaaledort** [comment]
I'm not sure the right thing to do is to keep the equals func in the commatrix repo instead of just use in the test.
It's an unused func in the library and makes an extra dependence between the test and the lib, for example if we want to ignore some ports in the test diff we will have to make a change in the matrix repo as well

**Review by aabughosh — APPROVED** [review]

**Comment by sabinaaledort** [comment]
I think using addFromFile func for the test is an interesting idea but I think you should consider to compare the csv files directly, it's an an extra dependence that might cause bugs. For example if we have a bug in the addFromFile func and we miss a port the test will fail...

**Pull request closed without merging**